index.html

```diff
@@ -80,13 +81,24 @@ <h1 class="title p-name" id="title">Requirements for Japanese Text Layout<br/><s
     <p its-locale-filter-list="ja" lang="ja">指摘を追跡しやすくするため，それぞれの指摘点について対象となるセクションを明記のうえ，個別の issue もしくはメールを送ってください．</p>
   </div>
 
-
   <p id="langSwitch">
     <button onclick="switchLang('ja')">日本語</button>
     <button onclick="switchLang('en')">English</button>
     <button onclick="switchLang('all')">All</button>
   </p>
 
+<p its-locale-filter-list="ja" lang="ja">
+この文書は<a href="https://www.w3.org/International/core/">国際化ワーキンググループ</a>によりワーキンググループノートとして公開されています．
+</p><p its-locale-filter-list="ja" lang="ja">
+この文書に関する議論には <a href="https://github.com/w3c/jlreq/issues/">GitHub Issues</a> の利用が望まれます．
+</p><p its-locale-filter-list="ja" lang="ja">
+ワーキンググループノートとしての公開は <abbr title="World Wide Web Consortium">W3C</abbr> 会員による支持を意味しません．この文書は草案であり，更新やほかの文書により置き換えられる可能性が常にあります．作業版以外としての参照は推奨されません．
+</p><p its-locale-filter-list="ja" lang="ja" data-deliverer="32113">
+この文書は<a href="https://www.w3.org/Consortium/Patent-Policy/"><abbr title="World Wide Web Consortium">W3C</abbr> 特許ポリシー</a>に従って運営されているグループにより作成されました．このグループはこの文書を <abbr title="World Wide Web Consortium">W3C</abbr> 勧告とすることを想定していません．
```

Comment:
Should we also translate the "W3C maintains a public list of any patent disclosures..." part?

Reply:
That will not be included under WG NOTE configuration.

Reply:
I see. It's not in https://w3c.github.io/jlreq/index.html?specStatus=WG-NOTE indeed.